### PR TITLE
ElasticSearch aufräumen/verbessern

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: app/grails-app/domain/de/laser/FTControl.groovy [hunk 1]
```diff
@@ -8,7 +8,7 @@ class FTControl {
     Integer esElements
     Integer dbElements
 
-    boolean active = true
+    boolean active
```
Comment (croqueGrec09) [inline app/grails-app/domain/de/laser/FTControl.groovy:11]: Boolean values should have a preset value, i.e. true or false

File: app/grails-app/services/com/k_int/kbplus/ESSearchService.groovy [hunk 2]
```diff
@@ -236,34 +258,33 @@ class ESSearchService{
...
           }
           catch ( Exception e ) {
-            log.error("Problem procesing mapping, key is ${mapping.key} value is ${params[mapping.key]}",e);
+            log.error("Problem procesing mapping, key is ${mapping.key} value is ${params[mapping.key]}",e)
           }
         }
       }
     }
 
     if(params?.searchObjects && params?.searchObjects != 'allObjects'){
```
Comment (croqueGrec09) [inline app/grails-app/services/com/k_int/kbplus/ESSearchService.groovy:286]: params?

File: app/grails-app/services/de/laser/DeletionService.groovy [hunk 3]
```diff
@@ -924,8 +924,10 @@ class DeletionService {
         RestHighLevelClient esclient = ESWrapperService.getClient()
 
         try {
-            DeleteRequest request = new DeleteRequest(es_index, id)
-            DeleteResponse deleteResponse = esclient.delete(request, RequestOptions.DEFAULT);
+            if(ESWrapperService.testConnection()) {
+                DeleteRequest request = new DeleteRequest(es_index, id)
+                DeleteResponse deleteResponse = esclient.delete(request, RequestOptions.DEFAULT);
```
Comment (croqueGrec09) [inline app/grails-app/services/de/laser/DeletionService.groovy:929]: ;

**Review by croqueGrec09 — COMMENTED** [review]
remarks

**Review by croqueGrec09 — APPROVED** [review]
approved

**Review by croqueGrec09 — APPROVED** [review]
update approved

**Review by klodav — APPROVED** [review]

**Pull request merged**